settings.gradle

```diff
@@ -8,20 +8,21 @@ include 'FixedSizeTextColumns'
 include 'Fragments'
 include 'GsonJsonWebservice'
 include 'LeftNavBarLibrary'
-project(':LeftNavBarLibrary').projectDir = new File('LeftNavBarExample/LeftNavBarLibrary') 
+project(':LeftNavBarLibrary').projectDir = new File('LeftNavBarExample/LeftNavBarLibrary')
 include 'LeftNavBarExample'
-project(':LeftNavBarExample').projectDir = new File('LeftNavBarExample/Example') 
+project(':LeftNavBarExample').projectDir = new File('LeftNavBarExample/Example')
 include 'ListStyles'
 include 'LivewallpaperSwitchAnimation'
-project(':LivewallpaperSwitchAnimation').projectDir = new File('Livewallpaper/switch_animation') 
+project(':LivewallpaperSwitchAnimation').projectDir = new File('Livewallpaper/switch_animation')
 include 'LivewallpaperSwitchLeftRight'
-project(':LivewallpaperSwitchLeftRight').projectDir = new File('Livewallpaper/switch_leftright') 
+project(':LivewallpaperSwitchLeftRight').projectDir = new File('Livewallpaper/switch_leftright')
 include 'MapApiV2Demo'
 include 'MultipleContacts'
 include 'OptionalDependencies'
 include 'PinchZoomDetector'
 include 'simperAudioStreamer'
 include 'sms'
+include 'spritz'
```

Comment:
Please let's not do this. There's literally no reason to include this in the root project. It's an anti-pattern coming from historical reasons, that we should not adopt going forward (happens the same with the dojo repo).

Reply:
I was merely adhering to what I read in there 😂 
Done in 96a580d7747ed633d2e2e0672dee327caaeb826c!